cubewalkers/parser.py

```diff
@@ -169,3 +170,99 @@ def bnet2rawkernel(rules: str,
     cpp_body += '\n} else{A__reserved_output[a__reserved]=A__reserved_input[a__reserved];}}}'
 
     return cp.RawKernel(cpp_body, kernel_name), varnames, cpp_body
+
+def network_rules_from_cana(N: BooleanNetwork) -> str:

```

Comment:
Please do not use `N` for the name of the `BooleanNetwork` because we use it elsewhere to denote the number of nodes.

Reply:
Changed it to BN